easybuild/easyconfigs/l/lavaan/lavaan-0.6-2-intel-2018a-R-3.4.4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'RPackage'
+
+name = 'lavaan'
+version = '0.6-2'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://cran.r-project.org/web/packages/%(name)s'
+description = """Fit a variety of latent variable models, including confirmatory factor analysis, structural
+ equation modeling and latent growth curve models."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [
+    'https://cran.r-project.org/src/contrib/',
+    'https://cran.r-project.org/src/contrib/Archive/$(name)s/',
```

Comment:
this should be `%(name)s`

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/6856/commits/4c9391d74a57d43ff95a7f77c651547aa144f461